scout/build/variant/variant.py

```diff
@@ -210,13 +210,13 @@ def build_variant(
     variant_obj["mei_polarity"] = variant.get("mei_polarity")
 
     ## Fusion variant specific
-    FUSION_KEYS = ["tool_hits", "fusion_score", "orientation", "frame_status", "fusion_genes"]
```

Comment:
I suspect that we are currently saving these specific key/values in all parsed values at the moment. I missed this during the review of the fusion variants PR! :disappointed:

Reply:
Oopsie! Good that you saw it now!